src/css/fonts.css

```diff
@@ -0,0 +1,41 @@
+@charset "UTF-8";
+
+@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,300i,400,400i,600,600i,700,700i,900,900i");
+@import url("https://fonts.googleapis.com/css?family=Source+Serif+Pro:400,600,700");
```

Comment:
Do we still need this Source Serif Pro line now that we have the one below at line 8? Looks like that one includes both regular and italic.

Reply:
Guess not.
```suggestion
```